autorag/nodes/passagefilter/recency.py

```diff
@@ -0,0 +1,50 @@
+import logging
+from datetime import datetime
+from typing import List, Tuple
+
+from autorag.nodes.passagefilter.base import passage_filter_node
+
+logger = logging.getLogger("AutoRAG")
+
+
+@passage_filter_node
+def recency_filter(contents_list: List[List[str]],
+                   scores_list: List[List[float]], ids_list: List[List[str]],
+                   time_list: List[List[datetime]],
+                   threshold: str,
```

Comment:
Why don't you get strptime type as parameter? Sometimes user wants to set threshold without hour:minute:seconds, sometimes day want to do.